_sass/3-layout/_global.scss

```diff
@@ -0,0 +1,103 @@
+@import "./0-base/reset";
+@import "./1-modules/mixins";
+@import "./1-modules/colors";
+@import "./1-modules/selectBox";
+@import "./1-modules/stickyAside";
+
+@import "./list";
+@import "./email";
+
+body {
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -moz-font-feature-settings: "kern", "liga", "clig", "calt";
+  -ms-font-feature-settings: "kern", "liga", "clig", "calt";
+  -webkit-font-feature-settings: "kern", "liga", "clig", "calt";
+  font-feature-settings: "kern", "liga", "clig", "calt";
+  background-color: var(--bg);
+  font-family: 'Source Code Pro', 'Helvetica', 'Arial', sans-serif;
+  font-size: 16px;
+  line-height: 1.6;
+  font-weight: 400;
+  color: var(--text);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  border-bottom: 1px solid var(--link_subtle);
+
+  &:hover {
+    border-color: var(--subtle_);
+  }
+
+  &:focus {
+    outline: none;
+    border-bottom: 1px solid var(--text_subtle);
+  }
+}
+
+em,
+i {
+  font-weight: 400;
+  font-style: italic;
+}
+
+b,
+strong {
+  font-weight: 700;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 1.2;
+
+  @include breakpoint(s) {
+    font-size: 30px;
+  }
+}
+
+h2 {
+  font-size: 24px;
+  line-height: 1.2;
+  font-weight: 400;
+  margin-top: $x_2;
+
+  @include breakpoint(s) {
+    font-size: 22px;
+  }
+}
+
+header,
+footer {
+  @include siteWidth;
+}
+
```

Comment:
Using the general reset left no room between paragraphs in body text! 

<img width="345" alt="Screen Shot 2020-06-12 at 4 56 20 PM" src="https://user-images.githubusercontent.com/221550/84545405-383cc000-accd-11ea-8d7c-a91c094f0918.png">

```suggestion
p {
  margin-bottom: $x_2;
}
```

orrrr

```
p + p {
  margin-top: $x_2;
}
```